cpp/client.hpp

```diff
@@ -199,11 +199,11 @@ class Client
   //! Locally stored io service.
   boost::asio::io_service io_service;
 
-  deadline_timer deadline;
-
   //! Locally stored socket object.
   tcp::socket s;
 
+  deadline_timer deadline;
```

Comment:
```suggestion
  //! Object to control connection timeouts.
  deadline_timer deadline;
```

Just a minor addition.